randomizer/Patching/CosmeticColors.py

```diff
@@ -2531,6 +2556,26 @@ def placeKrushaHead(slot):
     """Replace a kong's face with the Krusha face."""
```

Comment:
Forgot the colorblind mode check here, see PR #2204

Reply:
Figures I'd forget something. Adding it back.